ddtrace/span.py

```diff
@@ -486,14 +488,13 @@ def pprint(self):
             ("resource", self.resource),
             ("type", self.span_type),
             ("start", self.start),
-            ("end", "" if not self.duration else self.start + self.duration),
-            ("duration", "%fs" % (self.duration or 0)),
+            ("end", None if not self.duration else self.start + self.duration),
+            ("duration", self.duration),
             ("error", self.error),
-            ("tags", ""),
+            ("tags", ",".join("%s:%s" % (str(k), v) for (k, v) in sorted(self.meta.items()))),
+            ("metrics", ",".join("%s:%s" % (str(k), v) for (k, v) in sorted(self.metrics.items()))),
         ]
-
-        lines.extend((" ", "%r:%s" % kv) for kv in sorted(self.meta.items()))
-        return "\n".join("%10s %s" % line for line in lines)
+        return json.dumps(collections.OrderedDict(data))
```

Comment:
One concern here is that this could possibly raise if some bad data is set on a span.

eg:

```
>>> span.service = object()
>>> span.pprint()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "ddtrace/span.py", line 497, in pprint
    return json.dumps(collections.OrderedDict(data))
  File "/usr/local/Cellar/python@2/2.7.17_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/__init__.py", line 244, in dumps
    return _default_encoder.encode(obj)
  File "/usr/local/Cellar/python@2/2.7.17_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 207, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/local/Cellar/python@2/2.7.17_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 270, in iterencode
    return _iterencode(o, 0)
  File "/usr/local/Cellar/python@2/2.7.17_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 184, in default
    raise TypeError(repr(o) + " is not JSON serializable")
TypeError: <object object at 0x101237870> is not JSON serializable
```

Maybe we should convert all the values using `six.text_type` before dumping to json?